python/setup.py

```diff
@@ -0,0 +1,24 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="aswfdocker",
+    version="0.1.0",
+    author="Aloys Baillet",
+    author_email="aloys.baillet+github@gmail.com",
```

Comment:
Not a comment about this repo specifically, but at some point we should figure out if it would make sense to have a "global" author email for ASWF projects to ensure continuity when someone transitions out of the organization.